tsl/src/continuous_aggs/invalidation.c

```diff
@@ -250,19 +250,178 @@ cut_invalidation_along_refresh_window(const CaggInvalidationState *state, int64
 	return result;
 }
 
+#define IS_VALID_INVALIDATION(entry) ((entry)->hyper_id > 0)
+
+static void
+invalidation_entry_reset(Invalidation *entry)
+{
+	MemSet(entry, 0, sizeof(Invalidation));
+}
+
+#define invalidation_entry_set(entry, ti, hypertable_id, type)                                     \
+	do                                                                                             \
+	{                                                                                              \
+		bool should_free;                                                                          \
+		HeapTuple tuple = ts_scanner_fetch_heap_tuple(ti, false, &should_free);                    \
```

Comment:
I didn't change this because I wasn't sure why you want only some macro parameters to be all caps. Also, I haven't seem many other places where macro parameters are all caps.

Reply:
I actually prefer to have all parameters as caps, but I only changed those that had matching changes.

Granted, this convention does not seem to be used in the PG code, so this is a convention that exists in other guidelines to distinguish parameter names. If you decide to not change the parameters, I'm fine since this is not something that we have as a guideline.

Please add the parentheses though.